torchbenchmark/models/densenet121/__init__.py

```diff
@@ -56,7 +54,7 @@ def train(self, niter=3):
 
     def eval(self, niter=1):
         model = self.eval_model
-        example_inputs = self.example_inputs
+        example_inputs = self.infer_example_inputs
         example_inputs = example_inputs[0]
```

Comment:
Should we remove this line because it only picks the first element in `infer_exmaple_inputs`? For example, after this line, `example_inputs.shape == [3, 224, 224]` instead of `[eval_bs, 3, 224, 224]`?

Reply:
Sure, I will remove that. It's strange that increasing bs showed different utilizations when I collected the profiles and timestamps for eval.

Reply:
after I removed this, I am getting an error:
```
TypeError: conv2d() received an invalid combination of arguments - got (tuple, Parameter, NoneType, tuple, tuple, tuple, int), but expected one of:
 * (Tensor input, Tensor weight, Tensor bias, tuple of ints stride, tuple of ints padding, tuple of ints dilation, int groups)
      didn't match because some of the arguments have invalid types: (!tuple!, !Parameter!, !NoneType!, !tuple!, !tuple!, !tuple!, int)
 * (Tensor input, Tensor weight, Tensor bias, tuple of ints stride, str padding, tuple of ints dilation, int groups)
      didn't match because some of the arguments have invalid types: (!tuple!, !Parameter!, !NoneType!, !tuple!, !tuple!, !tuple!, int)
```

Reply:
I see that actually this line was wrapped in a tuple:
```
        self.example_inputs = (torch.randn((train_bs, 3, 224, 224)).to(self.device),)
        self.infer_example_inputs = (torch.randn((eval_bs, 3, 224, 224)).to(self.device),)
```

Reply:
Found a fix!